src/vsphere_cpi/spec/integration/vm_host_rule_spec.rb

```diff
@@ -45,15 +48,19 @@
       cpi.client.cloud_searcher.get_managed_object(
         VimSdk::Vim::ClusterComputeResource, name: @second_cluster_name)
     end
-    let(:first_host_vm_group_name) { @first_host_group+VSphereCloud::Resources::Cluster::CLUSTER_VM_GROUP_SUFFIX }
-    let(:second_host_vm_group_name) { @second_host_group+VSphereCloud::Resources::Cluster::CLUSTER_VM_GROUP_SUFFIX }
-    let(:third_host_vm_group_name) { @third_host_group+VSphereCloud::Resources::Cluster::CLUSTER_VM_GROUP_SUFFIX }
+    let(:first_host_vm_group_name) { @first_host_group+'MUST'+VSphereCloud::Resources::Cluster::CLUSTER_VM_GROUP_SUFFIX }
```

Comment:
Can we use string interpolation here instead of addition? This is kind of hard to read.

Reply:
I shortened the constant with Another `let` but on changing it I believe `+` is neater than interpolation via `"#{something}..."`